x/ccv/provider/keeper/relay_test.go

```diff
@@ -68,11 +68,14 @@ func TestQueueVSCPackets(t *testing.T) {
 		mocks := testkeeper.NewMockedKeepers(ctrl)
 		testkeeper.SetupMocksForLastBondedValidatorsExpectation(mocks.MockStakingKeeper, 0, []stakingtypes.Validator{}, 1)
 
+		mocks.MockStakingKeeper.EXPECT().GetBondedValidatorsByPower(gomock.Any()).Return([]stakingtypes.Validator{}, nil).AnyTimes()
```

Comment:
Is this indented correctly? Looks like it's fully on the left in comparison to the others.

Reply:
Looks fine to me.

Reply:
also fine for me, but gets confusing/wrong when I scale down the browser window